plugins/autocomplete/plugin.js

```diff
@@ -718,36 +718,30 @@
 		 * For example: the position of the bottom end of the caret.
 		 */
 		setPosition: function( rect ) {
-			var window = this.document.getWindow(),
-				panelHeight = this.element.getSize( 'height' ),
-				viewPane = window.getViewPaneSize(),
-				viewScroll = window.getScrollPosition(),
-				// Position of the top and bottom boundaries of the view
-				// relative to the body.
-				viewTop = viewScroll.x,
-				viewBottom = viewScroll.x + viewPane.height,
-				// How much space is there for the panel above and below the specified rect.
-				spaceAbove = rect.top - viewTop,
-				spaceBelow = viewBottom - rect.bottom,
-
-				frame = this.editor.window.getFrame(),
-				editable = this.editor.editable(),
+			var editor = this.editor,
+				viewPanelHeight = this.element.getSize( 'height' ),
+				windowFrame = editor.window.getFrame(),
+				editable = editor.editable(),
 				// Bounding rect where view should fit (visible editor viewport).
-				absoluteRect = editable.isInline() ? editable.getAbsoluteClientRect( this.editor ) : frame.getAbsoluteClientRect( this.editor ),
+				editorViewportRect = editable.isInline() ? editable.getAbsoluteClientRect( editor ) : windowFrame.getAbsoluteClientRect( editor ),
+				// How much space is there for the panel above and below the specified rect.
+				spaceAbove = rect.top - editorViewportRect.top,
+				spaceBelow = rect.bottom - editorViewportRect.bottom,
```

Comment:
Shouldn't the space below be computed by `editorViewportRect.bottom` - `rect.bottom`? Surprisingly I don't see any visible issues related to this value being computed incorrectly.